server/app/views/BaseHtmlView.java

```diff
@@ -50,6 +50,17 @@ public static DivTag fieldErrors(
         .withClasses(BaseStyles.FORM_ERROR_TEXT_BASE, StyleUtils.joinStyles(additionalClasses));
   }
 
+  /**
+   * Creates a button that doesn't display any text but includes an aria label, which is needed for
+   * accessibility.
+   *
+   * @param ariaLabel a label that will be used by screenreaders and other accessibility services to
+   *     describe the button's purpose.
+   */
+  public static ButtonTag noTextButton(String ariaLabel) {
```

Comment:
**suggestion**: would `iconButton` but a more descriptive name for this?

Reply:
I like `iconOnlyButton`, done.